config/secrets.yml

```
@@ -0,0 +1,8 @@
+development:
+  secret_key_base: b6d920b5cfb6ba56f674e66a97909044073a8ea0cfa95572b99d1e986eb38f8c3a915253a1266427a5eb48b0a7cbf4a53a073998ca10fe7fe062fd09dead6500
+
+test:
+  secret_key_base: 79b005cbc40f847db49a3437579f1746c86b18f28490ada333a9b3c687549fc07315d6dd1e42cff11e71f4d338ac2539a88cd042212436ee8dd6b10335ae0646
+
+production:
+  secret_key_base: cff8e2bda22d85d25cdd6c75cbbf56873230c320e1079f4cb4bc5a8255c617718d1eb48253bcc3317171a999b28870b216b7b818597ceb363653d80006f49a84
```

In production should be like this:
```suggestion
  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
```